src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck start work");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck stop work");

```

Comment:
Similarly to the doWork method, the stopWork message 'Truck stop work' should be more informative. Consider rephrasing it to 'Truck has stopped working.' to clearly indicate the state of the Truck.